### DISCUSSION TITLE
Split File Size -- Rclone

### DISCUSSION THREAD

**Discussion by wes1993** [tgdrive/teldrive#179] [Q&A] [answered]
Hello,
Thanks for your wonderful work @divyam234  
There is a way to change the Split File Size when i use rclone for upload? (Saving a file in the directory mounted with Rclone)

Best Regards
Stefano

**Comment by azeyrx** [comment]
[Rclone Config Example](https://github.com/divyam234/teldrive?tab=readme-ov-file#rclone-config-example)

`chunk_size = 500M`

**Reply by wes1993** [reply]
Thanks for your reply,
Changing this to 2000M means that i can upload file of 2Gb Is correct?

Best regards
Stefano

**Reply by azeyrx** [reply, marked as answer]
No, you can upload files of any size. These are only the chunks that are stored in Telegram, it is actually recommended to leave the size at 500M per chunk.

**Comment by wes1993** [comment]
Thanks for your reply :-D
